weasyprint/css/validation/properties.py

```diff
@@ -89,10 +89,11 @@ def validate_non_shorthand(base_url, name, tokens, required=False):
     if not required and name not in PROPERTIES:
         raise InvalidValues('property not supported yet')
 
-    for token in tokens:
-        var_function = check_var_function(token)
-        if var_function:
-            return ((name, var_function),)
+    if name != 'content':
```

Comment:
Why is this change needed? I’d really like to avoid special cases based on property name…